src/main/java/hyperledger/besu/java/rest/client/config/EthConfig.java

```diff
@@ -23,8 +23,9 @@ public class EthConfig {
 
   private List<Web3j> web3jList;
   @Autowired private EthProperties ethProperties;
-
-  @RefreshScope
+// Commenting this is a workaround for the application to bootstrap,
+// will remove it once we find a good fix for this
```

Comment:
@kamini-kamal PR has a workaround which still has @RefreshScope, do we want to move that approach. She is using a wrapper for Credentials

Reply:
Yes, I saw the approach, it seems good. Didn't add it here as it would be merged as part of her PR.

Reply:
Probably you can remove the comment

Reply:
Removed the comment